fabric-levelgenerators-v1/src/main/java/net/fabricmc/fabric/mixin/levelgenerator/AccessLevelGeneratorType.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.mixin.levelgenerator;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Invoker;
+
+import net.minecraft.world.level.LevelGeneratorType;
+
+@Mixin(LevelGeneratorType.class)
+public interface AccessLevelGeneratorType {
+	@Invoker("<init>")
+	static LevelGeneratorType fabric_init(int id, String name, String storedName, int version) {
```

Comment:
While it's an invoker for `<init>` it is *creating* a new instance of the object.
```suggestion
	static LevelGeneratorType fabric_create(int id, String name, String storedName, int version) {
```